src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -269,8 +269,6 @@ internal interface ISyntaxFactsService : ILanguageService
 
         bool IsAnonymousFunction(SyntaxNode n);
 
-        bool IsLocalFunction(SyntaxNode n);
```

Comment:
removed as there is already an IsLocalFunctionStatement.

Reply:
Wouldn't `IsLocalFunction` be preferable? The other one isn't `IsAnonymousFunctionExpression` but only `IsAnonymousFunction`.

Reply:
Yeah, this is my bad.  I've been a bit inconsistent here over hte lifetime of this type.  It would be nice to just make things much more consistent across the board here. My preference would be IsXXX  where XXX are the names you see in syntax as XXXSyntax.  i.e. if there is ParenthesizedExpressionSyntax, then the helper should be IsParenthesizedExpression and not IsParenthesized.  It's a simple and easy naming pattern that produces understandable results that are not too large.